nsv13/code/modules/overmap/types/syndicate.dm

```diff
@@ -429,6 +429,43 @@
 
 /obj/structure/overmap/syndicate/ai/kadesh/on_interdict()
 	add_sensor_profile_penalty(150, 10 SECONDS)
+	
+/obj/structure/overmap/syndicate/ai/fistofsol
+	name = "\proper SSV Fist of Sol"
```

Comment:
```suggestion
	name = "\improper SSV Fist of Sol"
```

Reply:
While @Bokkiewokkie was right in that this is not technically an improper noun. It needs to be referred as such for name references to work correctly, also \proper is implied when the name is capitalized.

proper:
`SSV Fist of Sol is hit by x`
improper:
`The SSV Fist of Sol is hit by x`